docs/tools/lsp-factoryjs/deployment/03-reactive-deployment.md

```diff
@@ -0,0 +1,242 @@
+---
+sidebar_position: 3
+title: Reactive Deployment
+---
+
+## Reactive Deployment
+
+The `lsp-factory.js` uses the [RxJS](https://github.com/ReactiveX/rxjs) library to deploy contracts. Developers can leverage this to achieve reactive deployment of Universal Profiles and Digital Assets.
+
+When deploying a Universal Profile or Digital Asset, pass the `deployReactive` flag inside the [ContractDeploymentOptions](../deployment/05-contract-deployment-options.md) object to receive an RxJS Observable, which will emit events as your contract is deployed.
+
+### Universal Profiles
+
+Use the `deployReactive` flag and the `subscribe()` function to listen for deployment events.
+
+```typescript
+let deploymentEvents = [];
+
+lspFactory.LSP3UniversalProfile
+  .deploy({...}, { deployReactive: true })
+  .subscribe({
+    next: (deploymentEvent) => {
+      deploymentEvents.push(deploymentEvent);
+    },
+    complete: () => {
+      console.log(deploymentEvents);
+    },
+  });
+```
+
+The function defined in `next` will be called whenever a new deployment event is created. We are simply pushing every deployment event into a `deploymentEvents` array.
+
+The function defined in `complete` will be called once after the deployment is finished. The following code snippet describes, how we can log the `deploymentEvents` array.
+
+```typescript title="Output from calling the console.log(deploymentEvents) function"
+[
+  {
```

Comment:
These deployment events were formatted differently before. Normally we want to explode brackets as you have here but I think in this case it makes sense to go with the previous formatting as better allows the reader to view and compare the deployment events